packages/block-library/src/query/index.js

```diff
@@ -22,3 +24,7 @@ export const settings = {
 	variations,
 	deprecated,
 };
+
+// Importing this file includes side effects and is whitelisted
+// in block-library/package.json under `sideEffects`.
+addFilter( 'editor.BlockEdit', 'core/query', queryInspectorControls );
```

Comment:
Why do we build this as a hook and not directly in the edit function of the block?

Reply:
This is a temporary solution.

>I have added a filter to Query edit in order to show the create new post link at the top of the Inspector controls. The order of InspectorControl children is based on the render order and with the addition of the new supports(color, layout) it was not displayed properly. The proper solution will be with an implementation of grouping these controls like the recent change in BlockControls with the group key.

Alternatively we could create a new `slot` under `BlockCard` component for such use cases. I think I've seen some designs that want to add things right below BlockCard info.